g:    9.81
mass: 0.98
Ixx:  2.64e-3
Iyy:  2.64e-3
Izz:  4.96e-3
k_f:  2.0e-8
k_m:  0.01302 # 0.01302

time_constant: 0.01

THRUST_NOISE_MEAN:  0.0
THRUST_NOISE_COV:   0.0
ANGULAR_SPEED_MEAN: 0.0
ANGULAR_SPEED_COV:  0.0 

RADIUS:     1.5
LINEAR_VEL: 3

# POSITION MEASUREMENT COVARIANCE FROM VICON
POS_MEAS_MEAN:  0.00

POS_MEAS_COV:   0.001 # 0.20
VEL_MEAS_COV:   0.001 #
ROT_MEAS_COV:   0.001 #
OME_MEAS_COV:   0.001 # 0.059 deg/s

# POS_MEAS_COV:   0.001 # 0.20
# VEL_MEAS_COV:   0.001 #
# ROT_MEAS_COV:   0.001 #
# OME_MEAS_COV:   0.001 # 0.059 deg/s

# Drag force 
DRAG_FORCE_X:   -0.3
DRAG_FORCE_Y:   -0.3
DRAG_FORCE_Z:   -0.2
# Drag force 
# DRAG_FORCE_X:   -0.20
# DRAG_FORCE_Y:   -0.23
# DRAG_FORCE_Z:   -0.30
TRJ_LEN_MAX:     10.0
OBS_NUM:         6
OBS1_RADIUS:     0.08
SAFE_D:          0.00
# Driver delay (s)
DRIVER_DELAT:    0.05

# DRAG_FORCE_X:   0.2
# DRAG_FORCE_Y:   0.2
# DRAG_FORCE_Z:   0.2
# Environment Simulation
MAP_X:           5.0
MAP_Y:           5.0
MAP_Z:           2.0

MAP_CENTER_X:    0.0
MAP_CENTER_Y:    0.0
MAP_CENTER_Z:    1.0

LANDMARKS_SIZE:  1000
LIDAR_RANGE:     0.60
LIDAR_RANGE_MIN: 0.10

TEST_RECOVERY:   FALSE
MOVE_X:          0.00
MOVE_Y:          0.30
MOVE_Z:         -0.40

#Geometry
NUM_ROTORS:    4
ROTOR_POS_X1:  15
ROTOR_POS_Y1:  0
ROTOR_POS_Z1:  0
ROTOR_AXIS_X1: 0
ROTOR_AXIS_Y1: 0
ROTOR_AXIS_Z1: 1

ROTOR_POS_X2: -15
ROTOR_POS_Y2:  0
ROTOR_POS_Z2:  0
ROTOR_AXIS_X2: 0
ROTOR_AXIS_Y2: 0
ROTOR_AXIS_Z2: 1

ROTOR_POS_X3:  0
ROTOR_POS_Y3:  15
ROTOR_POS_Z3:  0
ROTOR_AXIS_X3: 0
ROTOR_AXIS_Y3: 0
ROTOR_AXIS_Z3: 1

ROTOR_POS_X4:  0
ROTOR_POS_Y4: -15
ROTOR_POS_Z4:  0
ROTOR_AXIS_X4: 0
ROTOR_AXIS_Y4: 0
ROTOR_AXIS_Z4: 1

ct: 0
km: 0
